learn/core_concepts/relevancy.md

```diff
@@ -12,20 +13,26 @@ In order to ensure relevant results, search responses are sorted based on a set
 
 Each index possesses a list of ranking rules stored as an array in the [settings object](/reference/api/settings.md). This array is **fully customizable**, meaning you can **delete existing rules, add new ones, and reorder them as needed**.
 
-Whenever a search query is made, Meilisearch uses a [bucket sort](https://en.wikipedia.org/wiki/Bucket_sort) algorithm to rank documents. The first ranking rule is applied to all documents, while each subsequent rule is only applied to documents that are considered equal under the previous rule (i.e. as a tiebreaker).
+Meilisearch uses a [bucket sort](https://en.wikipedia.org/wiki/Bucket_sort) algorithm to rank documents whenever a search query is made. The first ranking rule is applied to all documents, while each subsequent rule is only applied to documents that are considered equal under the previous rule (i.e., as a tiebreaker).
 
 **The order in which ranking rules are applied matters.** The first rule in the array has the most impact, and the last rule has the least. Our default configuration meets most standard needs but [you can change it](/reference/api/ranking_rules.md#update-ranking-rules).
 
 ### Built-in rules
 
 Meilisearch contains six built-in ranking rules in the following order:
 
-1. Words
-2. Typo
-3. Proximity
-4. Attribute
-5. Sort
-6. Exactness
+```json
+[
+  "words",
+  "typo",
+  "proximity",
+  "attribute",
+  "sort",
+  "exactness"
+]
+```
+
+Depending on your needs, you might want to change this order of importance. To do so, you can use the [update settings](/reference/api/settings.md#update-settings) or [update ranking rules endpoint](/reference/api/ranking_rules.md#update-ranking-rules).
```

Comment:
Minor cosmetic suggestion (remove "of importance", sentence seems to work fine without it) and making it clear that "update settings" is also an endpoint. Not necessarily pretty, but I think it's potentially confusing otherwise.

```suggestion
Depending on your needs, you might want to change this order. To do so, you can use the [update settings endpoint](/reference/api/settings.md#update-settings) or [update ranking rules endpoint](/reference/api/ranking_rules.md#update-ranking-rules).
```